.github/workflows/cleanup-all.yml

```diff
@@ -1,9 +1,14 @@
 name: Clean Atlas organization
 # local run sample:
-# echo '{"ref":"HEAD/ref/CLOUDP-smth", "ref_type":"branch"}' > payload.json | act -j cleanup-all -e payload.json
-# ^ command will clean branch organization with name `CLOUDP-smth`
+# echo '{"ref":"HEAD/ref/CLOUDP-smth", "ref_type":"branch", "action":"workflow_dispatch", "inputs": {"clean":"true"} }' > payload.json | act -j cleanup-all -e payload.json
+
 on:
   workflow_dispatch:
+    inputs:
+      clean:
+        description: "Clean all(true) or old empty projects(false)?"
```

Comment:
I'm not sure how to explain it better in one line, this is ok? 
note: I believe for now better to live both cases before updating the ginkgo version and make sure after each is working. And after we can delete choses

Reply:
I think it's okay to leave it doing both still! I think it's clear enough what true & false do in this message.